### DISCUSSION TITLE
Feature Request : adding comments to DB schema / Entity

### DISCUSSION THREAD

**Discussion by HRR1337** [TiagoSilvaPereira/vemto-issues#213] [Ideas & Suggestions]
What really would be handy is to add an commentfield on a entity or on general db schema creater where I can put some notes:

For example: 
- Thought about why I have chosen about some structure / entity
- Links to best practices / tutorials etc. 

![comment](https://user-images.githubusercontent.com/4428675/129160982-918a2387-98b8-4a44-a3e5-82f604dc5e8a.jpg)

**Comment by TiagoSilvaPereira** [comment]
Thank you for your suggestion @HRR1337 it is already on my plans
